components/brave_rewards/browser/rewards_notification_service_impl.cc

```diff
@@ -297,14 +297,29 @@ void RewardsNotificationServiceImpl::OnGetAllNotifications(
   TriggerOnGetAllNotifications(rewards_notifications_list);
 }
 
+std::string RewardsNotificationServiceImpl::GetGrantIdPrefix(
+    const std::string& grant_type) {
+  std::string prefix = grant_type == "ugp"
```

Comment:
grant_type == "ugp" should be a method to remove duplication

Reply:
done